prboom2/src/m_menu.c

```diff
@@ -5823,6 +5828,31 @@ void M_StartControlPanel (void)
   itemOn = currentMenu->lastOn;   // JDC
 }
 
+
+/////////////////////////////////////////////////////////////////////////////
+//
+// Menu Shaded Overlay Stuff
+//
+// This displays a dark overlay under certain screens of the menus
+
+int screenshade = 20;
```

Comment:
We should use `static const` or macro if it is a constant. Do we need this variable?

Reply:
So in Nyan Doom, I have extra code where this variable is not constant... And so I was trying to keep the code close to the same between the two ports, while deleting extra animation fade code for DSDA Doom.

Reply:
Are there plans to merge code from Nyan into DSDA-Doom or vice versa?

Reply:
No, but isn't it better to keep the implementation of the same feature in both source ports as close as possible? As in if I add something to Nyan that DSDA wants in the future, it will be easier if DSDA is already using a very similar structure.

I mention this because there are certain things I can "optimise / simplify", but I haven't because I wanted to keep the structure relatively close to Nyan for consistency sake.